### PR TITLE
updated get quantile function

### PR DESCRIPTION
**Pull request by SHAESEN2**

### CODE REVIEW

File: tests/testthat/test-get_quantile.R [hunk 1]
```diff
@@ -1,7 +1,188 @@
...
+testthat::test_that("T6.3 The output contains a column with the quantities",{
+
+  survfit_object <- visR::estimate_KM(adtte, strata = "TRTA")
+  quant <- visR::get_quantile(survfit_object, conf.int = TRUE)
+  
+  quaNm <- as.factor(c("lower", "quantile", "upper"))
+  
+  testthat::expect_equal(quaNm, unique(quant[["quantity"]]))
+})
+
```
Comment (rebecca-albrecht) [inline tests/testthat/test-get_quantile.R:176]: Here:
-- Failure (Line 8): T6.3 The output contains a column with the quantities -------------------------
`quaNm` not equal to unique(quant[["quantity"]]).
'current' is not a factor

**Review by rebecca-albrecht — CHANGES_REQUESTED** [review]

File: tests/testthat/test-get_quantile.R [hunk 2]
```diff
@@ -1,7 +1,188 @@
...
+testthat::test_that("T6.2 The output contains a column with the strata names",{
+
+  survfit_object <- visR::estimate_KM(adtte, strata = "TRTA")
+  quant <- visR::get_quantile(survfit_object, conf.int = TRUE)
+  
+  stratNm <- as.factor(unique(names(survfit_object[["strata"]])))
+  
+  testthat::expect_equal(stratNm, unique(quant[["strata"]]))
+})
```
[the remark this answers is not in the document]
Comment (SHAESEN2) [reply tests/testthat/test-get_quantile.R:165]: image

File: tests/testthat/test-get_quantile.R (hunk 1 above)
Comment (SHAESEN2) [reply tests/testthat/test-get_quantile.R:176]: image

Comment (SHAESEN2) [reply tests/testthat/test-get_quantile.R:176]: Hey Rebecca, what is current referring to in your fail message? Are both objects factors for you?

Comment (rebecca-albrecht) [reply tests/testthat/test-get_quantile.R:176]: Hi Steven, this is not a factor: quant[["quantity"]]

Comment (SHAESEN2) [reply tests/testthat/test-get_quantile.R:176]: Can you reinstall this branch and try again? I think you might have the old function still in your session. There I am getting the same error.

Comment (rebecca-albrecht) [reply tests/testthat/test-get_quantile.R:176]: Still there

Comment (SHAESEN2) [reply tests/testthat/test-get_quantile.R:176]: Please try again. I made everything a character :(

Comment (rebecca-albrecht) [reply tests/testthat/test-get_quantile.R:176]: Now it works.

**Review by rebecca-albrecht — APPROVED** [review]

**Pull request merged**